services/shuttle-axum/src/lib.rs

```diff
@@ -40,5 +40,22 @@ impl<S> From<axum::Router<S>> for AxumService<S> {
     }
 }
 
-/// The return type that should be returned from the [shuttle_runtime::main] function.
+/// Return type from the `[shuttle_runtime::main]` macro for a Axum-based service.
+///
+/// # Example
+///
+/// ```rust,no_run
+/// use axum::{routing::get, Router};
+///
+/// async fn hello_world() -> &'static str {
+///     "Hello, world!"
+/// }
+///
+/// #[shuttle_runtime::main]
+/// async fn axum() -> shuttle_axum::ShuttleAxum {
+///     let router = Router::new().route("/hello", get(hello_world));
```

Comment:
Since we have done this in our examples, should we do it here though (for all these docs examples)?
```suggestion
///     let router = Router::new().route("/", get(hello_world));
```